tikv: fixed some description about `--db` and `compact-cluster` (#14576)
pingcap/docs: tikv-control.md

- To compact data in the local mode, use the following command:

    ```shell

>     tikv-ctl --data-dir /path/to/tikv compact --d kv

Author: ```suggestion
    tikv-ctl --data-dir /path/to/tikv compact -d kv
```

    ```

- To compact data in the remote mode, use the following command:

    ```shell

>     tikv-ctl --host ip:port compact --d kv

Author: ```suggestion
    tikv-ctl --host ip:port compact -d kv
```

    ```

### Compact data of the whole TiKV cluster manually

Use the `compact-cluster` command to manually compact data of the whole TiKV cluster. The flags of this command have the same meanings and usage as those of the `compact` command. The only difference is as follows:

> - For the `compact-cluster` command, you use `--pd` to specify the address of the PD as the compact target.

Reviewer: ```suggestion
- For the `compact-cluster` command, you use `--pd` to specify the address of the PD so `tikv-ctl` will be able to find all TiKVs in the cluster as compact target.
```

I think the target of compaction should be TiKV nodes, PDs are just metadata provider.

- To compact data in the local mode, use the following command:

    ```shell
    tikv-ctl --data-dir /path/to/tikv compact -d kv
    ```

- To compact data in the remote mode, use the following command:

    ```shell
    tikv-ctl --host ip:port compact -d kv
    ```

### Compact data of the whole TiKV cluster manually

Use the `compact-cluster` command to manually compact data of the whole TiKV cluster. The flags of this command have the same meanings and usage as those of the `compact` command. The only difference is as follows:

> - For the `compact-cluster` command, you use `--pd` to specify the address of the PD, so that `tikv-ctl` can locate all TiKV nodes in the cluster as the compact target.

Reviewer 2: ```suggestion
- For the `compact-cluster` command, use `--pd` to specify the address of the PD, so that `tikv-ctl` can locate all TiKV nodes in the cluster as the compact target.
```

> - For the `compact` command, you use `--data-dir` or `--host` to specify a single TiKV.

Reviewer 2: ```suggestion
- For the `compact` command, use `--data-dir` or `--host` to specify a single TiKV as the compact target.
```
